### ISSUE TITLE
a question about the backbone of covnext-xl

### ISSUE THREAD

**Issue by bartbuaa** [IDEA-Research/DINO#132]
It seems that various backbones obviously affect final MAP score.
Since ConvNeXt-XL surpass Swin-L on ImageNet-22k, it is curious that have you ever tried to utilize it for improving.
I notice that your code includes ConvNeXt-XL indeed. 
Could you offer some details about the MAP score on COCO with ConvNeXt-XL.

It is so appreciated!

**Comment by FengLi-ust** [comment]
In our preliminary experiments, SwinL outperforms ConvNex. But we did not tune the hyperparameters, like the weight decay. Therefore, we are not sure about the performance of ConvNext.
